gettingstarted/InstallingEquella.md

```diff
@@ -3,39 +3,46 @@
 # Installing openEQUELLA
 
 ## Environment setup
-Assuming you are building and installing openEQUELLA in the same environment, the following should be setup.  Details can be found in the [openEQUELLA Dev Setup](https://github.com/equella/Equella#download-required-software) guide.
+The following dependencies are required to successfully install openEQUELLA.
 
-- Java 8 (Oracle)
-- ImageMagick
-- Libav
-- Database
+- [Java 8 JDK](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) 
 
-Needed if you are building your own install binary:
-- GIT
-- SBT
+- [ImageMagick](https://www.imagemagick.org/script/download.php)
 
-## Build the Installer
+- [LibAV](https://libav.org/download/) (**Note:** This is not a compulsory step if video file attachments will not be used in your
+  institution or you don’t require them to display in the Videos gallery view.)
 
-If you don't have / don't want to use a pre-built binary, you can build openEQUELLA from source yourself by following the instructions on the main [openEQUELLA repo](https://github.com/equella/Equella).
+- Database Server
 
-## Install openEQUELLA
-1. Unzip the openEQUELLA install zip binary.
-1. Run the GUI installer:
+- - PostgreSQL
```

Comment:
Got an extra bullet there.

Reply:
My markdown editor wasn't showing that extra bullet, will remove it.